pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: test
-version: 0.12.21
+version: 0.12.21-dev
```

Comment:
Ugh. Lots of silly mistakes. I promise I'll learn the Github workflow soon enough!